# yaw, roll, pitch's unit is in 0.1 degree
# yaw  [-3200, 3200]
# roll [-350, 350]
# pitch  [-900, 300]
# duration's unit in 0.1s 

int16 yaw
int16 roll
int16 pitch
int16 duration
# absolute = 1, incremental = 0
bool absolute_or_incremental
bool yaw_cmd_ignore
bool roll_cmd_ignore
bool pitch_cmd_ignore
---
bool result 
